src/main/java/frc/robot/subsystems/conveyor/commands/CargoIn.java

```diff
@@ -0,0 +1,45 @@
+package frc.robot.subsystems.conveyor.commands;
+
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Ports;
+import frc.robot.Constants;
+import frc.robot.subsystems.conveyor.Conveyor;
+
+public class CargoIn extends CommandBase {
+    private final Conveyor conveyor;
+    private final WPI_TalonSRX motor = new WPI_TalonSRX(Ports.Conveyor.AUX);
```

Comment:
I know you're not using this variable (which means it shouldn't be declared), but not only does declaring hardware components outside of a subsystem go against convention, the code will also just crash because you can't declare multiple devices with the same CAN ID.